src/main/java/org/waveywaves/jenkins/plugins/tekton/client/build/create/CreateRaw.java

```diff
@@ -48,12 +50,12 @@
 import java.util.logging.Level;
 import java.util.logging.Logger;
 
-@Symbol("createStep")
+@Symbol("createRaw")
```

Comment:
```suggestion
@Symbol("tectonCreateRaw")
```

Otherwise that might be confusing and conflicting with other plugins